src/media/js/elements/header.js

```diff
@@ -97,6 +97,24 @@ define('elements/header',
 
     el.MktHeaderChildElement = document.registerElement('mkt-header-child', {
         prototype: Object.create(HTMLElement.prototype, {
+            createdCallback: {
+                value: function() {
+                    if (this.isInput) {
+                        var input = this.input;
+                        var label = this.querySelector('label');
+                        var setInputWidth = function() {
+                            if (input.value) {
```

Comment:
Made it a ternery, I don't think it formats really well but it's the best I could come up with.